kong/plugins/proxy-cache/handler.lua

```diff
@@ -173,11 +176,28 @@ local function cacheable_response(conf, cc)
       return false
     end
 
+    local t, subtype, params = parse_mime_type(content_type)
     local content_match = false
     for i = 1, #conf.content_type do
-      if conf.content_type[i] == content_type then
-        content_match = true
-        break
+      local expected_ct = conf.content_type[i]
+      local exp_type, exp_subtype, exp_params = parse_mime_type(expected_ct)
+      if exp_type then
+        if (exp_type == "*" or (t and lower(t) == lower(exp_type))) and
+          (exp_subtype == "*" or (subtype and
+            lower(subtype) == lower(exp_subtype))) then
```

Comment:
I believe the `lower`s are not really needed here, as `parse_mime_type` will return lowercase types and subtypes now. But this is a minor concern.